apps/judicial-system/backend/src/app/modules/defendant/models/defendant.model.ts

```diff
@@ -125,4 +125,8 @@ export class Defendant extends Model {
   })
   @ApiProperty()
   verdictViewDate?: string
+
+  @Column({ type: DataType.BOOLEAN, allowNull: true })
+  @ApiPropertyOptional({ type: Boolean })
+  acceptCompensationClaim?: boolean
```

Comment:
Should this field not be added to the judicial-system-api defendant model as well?

Reply:
True, I added it in! Although we're not technically using it anywhere on the web yet I think we probably will need to use it in the front end at some point in the near future.